explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -84,7 +84,8 @@ object AladinCell extends ModelOptics {
         implicit val ctx = props.ctx
         $.setStateL(State.fov)(fov) >>
           UserTargetPreferencesUpsert
-            .updateFov[IO](props.uid, props.tid, fov.x)
+            // TODO Accept multiple Target ids.
```

Comment:
This is another area that is a bit problematic for the UI. It will be difficult to not "reset" the FOV when updating the observations in a target * group.

Reply:
Except that names are not unique. Even if the API is changed to require uniqueness within a target environment, I don't think they'll need to be unique within a program.